src/hotspot/share/gc/shenandoah/shenandoahVerifier.cpp

```diff
@@ -411,11 +411,12 @@ class ShenandoahGenerationStatsClosure : public ShenandoahHeapRegionClosure {
               "%s: generation (%s) used regions (" SIZE_FORMAT ") must equal regions that are in use (" SIZE_FORMAT ")",
               label, generation->name(), generation->used_regions(), stats.regions());
 
-    size_t capacity = generation->adjusted_capacity();
-    guarantee(stats.span() <= capacity,
-              "%s: generation (%s) size spanned by regions (" SIZE_FORMAT ") must not exceed current capacity (" SIZE_FORMAT "%s)",
-              label, generation->name(), stats.regions(),
-              byte_size_in_proper_unit(capacity), proper_unit_for_byte_size(capacity));
+// This check is disabled because of known issues with this feature. We expect this code to be updated by 05/2023.
```

Comment:
Let's link a JBS ticket or a  draft PR to this comment, so it can be tracked? I realize the draft PR will essentially remove this code, but we want a peg here to hang the proverbial hat on, that's all.

Reply:
In addition, let's list any known tests that fail as a result of this assertion (that would need a ticket to track the failures, I guess).

Reply:
Created https://bugs.openjdk.org/browse/JDK-8306024 to make sure we don't forget about this.